css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: #1996da;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #fff;
+}
+
+.title {
+  padding: 2rem;
+  padding-bottom: 1rem;
+  font-size: 1.5rem;
+  letter-spacing: 1px;
+  text-align: center;
+}
+
+section.all {
+  display: flex;
+  justify-content: space-between;
+  width: 60%;
+  margin: 0 auto;
+}
+
+.first,
+.second,
+.third {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 2rem;
+}
+
+img {
+  width: 3rem;
+}
+
+.three-pics {
+  background-color: #fff;
+  position: absolute;
+  border-radius: 50%;
+  padding: 1.5rem;
+}
+
+.text {
+  text-align: center;
+  position: relative;
+
+  /* border: 1px solid black; */
+  top: 90%;
```

Comment:
If you undo the `absolute` positioning above, this should no longer be necessary.